src/Adapter/MSTest.CoreAdapter/Execution/TypeCache.cs

```diff
@@ -645,6 +646,10 @@ private int GetTestTimeout(MethodInfo methodInfo, TestMethod testMethod)
 
                 return timeoutAttribute.Timeout;
             }
+            else if (globalTimeout > 0)
```

Comment:
Have we asked @pvlakshm about precedence order?

Reply:
What happens when we passing runsettings arguments through commandline??

Reply:
https://github.com/Microsoft/testfx-docs/pull/40